packages/terra-date-picker/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Renamed react-datepicker's `datepicker.scss` to `datepicker.module.scss`.
```

Comment:
nit : `datepicker.scss` to `react_datepicker.module.scss`.

Reply:
Changed here : https://github.com/cerner/terra-framework/pull/906/commits/e691b0726e4b96b9dbd12881a8b0d0e97ee79815